app/backend/src/couchers/constants.py

```diff
@@ -44,3 +46,27 @@
 ACTIVENESS_PROBE_TIME_REMINDERS = [timedelta(days=0), timedelta(days=4, hours=8)]
 # total time from initiation after which to expire the probe
 ACTIVENESS_PROBE_EXPIRY_TIME = timedelta(days=14)
+
+
+class RateLimitAction(Enum):
+    """Possible user actions which can be rate limited."""
+
+    host_request = "host request"
+    friend_request = "friend request"
+    chat_initiation = "chat initiation"
```

Comment:
I would like this in `models.py` given that it's a DB enum.

Simultaneously I would like `constants.py` to not contain any real imports...

The former I think is more important than the latter. Or I would suggest you just move the rate limit definitions into the rate limit file.py